cmd/root.go

```diff
@@ -71,7 +72,7 @@ func Execute() error {
 	log.SetFlags(0)
```

Comment:
Let's remove it from here.
The `pkg/logging` should take care of 
> // Don't prefix messages with data & time to improve readability.

Reply:
Removed.